Test/Framework/Graphics/TextureCubeTest.cs

```diff
@@ -177,6 +186,9 @@ public void GetAndSetDataDxtDontRoundWhenOutsideBounds(string texName, int bs)
 
         [TestCase("SampleCube64DXT1Mips", 8)]
         //[TestCase("SampleCube64DXT5Mips", 16)]
+#if DESKTOPGL
+        [Ignore("GetData is not yet implemented for compressed images")]
+#endif
```

Comment:
I should get rid of these ignores and see if it all check out since I implemented GetData for compressed textures in this too.